Project/Assets/ML-Agents/Examples/GridWorld/Scripts/GridAgent.cs

```diff
@@ -67,6 +84,18 @@ public override void WriteDiscreteActionMask(IDiscreteActionMask actionMask)
         }
     }
 
+    private void ProvideReward(GridGoal hitObject)
+    {
+        if (gridGoal == hitObject)
```

Comment:
I wonder if we should ignore collision with the wrong goal, rather than give it a negative reward.  This is what im doing in WallJump/Pushblock.

Reply:
Well, I kept the original reward function. I worry that doing what you say will make the task too easy? I'd be curious to see the Push/Jump task with the +1/-1 reward.